src/scss/abstracts/_button-classes.scss

```diff
@@ -0,0 +1,35 @@
+@import "../abstracts/button-mixins";
+
+.btn-close {
+  border: none;
+  width: 2.5rem;
+  height: 2.5rem;
+  background: url("/assets/images/icons/close.svg") center no-repeat;
+  background-size: 1.5rem;
+  will-change: transform;
+  transition: transform 0.2s ease-out;
+
+  &:hover {
+    transform: scale(1.15);
+  }
+}
+
+.btn {
+  @include btn;
+}
+.btn.btn-primary {
+  @include btn-primary($white, $university-green);
```

Comment:
Is there any issue with this variable not being defined? Or is it somewhere and I'm missing that?